evalrev

```diff
@@ -28,40 +23,24 @@ if test -d "${DIR}/.git"; then
         REVISION="${REVISION}_rev-$(git rev-parse --verify --short HEAD)"
         SCM="_rev-$(git rev-parse --verify --short HEAD)"
     fi
-elif test -d "${DIR}/.svn"; then
-    REVISION="_r$(svnversion $DIR 2> /dev/null | sed 's/[^0-9]*//g')"
-    SCM="_r$(svnversion $DIR 2> /dev/null | sed 's/[^0-9]*//g')"
-
-    if test x$REVISION = "x"; then
-        REVISION="_r$(svn info $DIR 2> /dev/null | grep -i revision | sed 's/[^0-9]*//g')"
-        SCM="_r$(svn info $DIR 2> /dev/null | grep -i revision | sed 's/[^0-9]*//g')"
-    fi
-
-    if test x$REVISION = "x"; then
-        if test -f "${DIR}/.svn/entries"; then
-            REVISION="_r$(cat ${DIR}/.svn/entries | grep -i revision | head -n 1 | sed 's/[^0-9]*//g')"
-            SCM="_r$(cat ${DIR}/.svn/entries | grep -i revision | head -n 1 | sed 's/[^0-9]*//g')"
-        fi
-    fi
-
-    if test x$REVISION = "x"; then
-        REVISION="_r1"
-        SCM="_r1"
-    fi
-elif [ -n "${GITHUB_SHA}" ]; then
-    SHORT_SHA_REVISION="$(echo ${GITHUB_SHA} | cut -b 1-8)"
+elif [ -n "${GITHUB_SHA:-}" ]; then
+    SHORT_SHA_REVISION="$(echo "$GITHUB_SHA" | cut -b 1-8)"
 
-    VERSION=$(cat ${DIR}/AC_VERSION 2>/dev/null)
+    VERSION=$(cat "$DIR/AC_VERSION" 2>/dev/null)
```

Comment:
why removed {}?

Reply:
I removed it because, despite what shellcheck may tell you, the {} is not necessary, it's just extra characters in the line.

I have a hypothesis that whoever wrote this originally thought that `${VAR}` was the same thing as `"$VAR"` for the purposes of quoting. It's an easy mistake to make.

Reply:
I see
https://www.gnu.org/software/bash/manual/html_node/Word-Splitting.html